.gitignore

```diff
@@ -1,5 +1,6 @@
 _cache
 _static
 _templates
+*.pyc
```

Comment:
Can you back out these unrelated changes so we can focus on just the urlopen fix?

Reply:
removed